config.sh

```diff
@@ -696,6 +718,13 @@ if [ "$PLATFORM" = "psp" -o "$PLATFORM" = "gp2x" \
 	OVERLAY="false"
 fi
 
+if [ "$PLATFORM" = "emscripten" ]; then
+	echo "Force-enabling OpenGL ES support; force-disabling unsupported GL renderers (Emscripten)."
+	GLES="true"
+	GL_FIXED="false"
+	OVERLAY="false"
```

Comment:
For the intended effect in the current `master` you'll want to change this from `OVERLAY` to `SOFTSCALE`.

Reply:
The reason for disabling "overlay1/overlay2" was simply that they did not function correctly and were not really worth the effort of attempting to fix. As "softscale" *does* work, I see no reason to disable it.